app/components/request_form/request_form.html.erb

```diff
@@ -14,6 +14,20 @@
         <% end %>
 
         <%= c 'field', object: @request, attr: :text do |field| %>
+          <div
+            class='RequestForm-messageTextAreaToolbar'
+            role='toolbar'
+            aria-label='Symbolleiste'
```

Comment:
not sure if Toolbar might be easier to understand or if that is just my missing knowledge :D

Reply:
I just translated it in Deepl cause I guess we want to make it a11y for German users? Or, we want to add this also to the translation files, I guess. Do you just call it toolbar in German?